### PR TITLE
completed

### PR DESCRIPTION
**Pull request by Serhii-Pradid**
Перевір будь-ласка

### CODE REVIEW

File: controllers/contacts-controller.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+// import Joi from "joi";
+import contactsService from "../models/contacts.js";
+import {ctrlWrapper} from "../decorators/index.js";
+import { HttpError } from "../helpers/index.js";
+
+/* const contactsAddSchema = Joi.object ({
+    name: Joi.string().required().messages({
+      "any.required": `missing required name field`,
```
Comment (ceREAL133) [inline controllers/contacts-controller.js]: Delete unnecessary comments

File: controllers/contacts-controller.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+       if(!result) {
+         throw HttpError(404, `Movie with id=${contactId} not found`);
+    }
+       res.json(result);
+    } 
+    
+
+ const add = async (req, res) => {
+      // const {error} = contactsAddSchema.validate(req.body);
+      // if(error) {
```
Comment (ceREAL133) [inline controllers/contacts-controller.js]: Comments

File: controllers/contacts-controller.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      //   throw HttpError(400, error.message)
+      // }
+       const result = await contactsService.addContact(req.body)
+       res.status(201).json(result)
+    } 
+      
+
+ const updateById = async (req, res) => {
+      // const {error} = contactsAddSchema.validate(req.body);
+      // if(error) {
```
Comment (ceREAL133) [inline controllers/contacts-controller.js]: Delete

File: server.js [hunk 4]
```diff
@@ -1,4 +1,6 @@
-const app = require('./app')
+// const app = require('./app')
```
Comment (ceREAL133) [inline server.js]: Comments

**Review by ceREAL133 — APPROVED** [review]

**Comment by Serhii-Pradid** [comment]
видалив

**Pull request merged**